### ISSUE TITLE
Support for SX1262

### ISSUE THREAD

**Issue by JJJS777** [LoRaMesher/LoRaMesher#12]
Hello (::

I am trying to implement a project with LoRaMesher. My esp32 has a SX1262 LoRa chip. I saw that you have tagged the SX126x in the project, but I can't find any implementation with it. Do I have to adapt the LoRaMesher to the SX1262 myself?

I have already adapted the counter example and get it to run, unfortunately only the reception of packets does not work yet.

Do you have a tip for me? 
Thanks for your help!

**Comment by Jaimi5** [comment]
Hi!

It is not yet implemented the support for SX126X but it should be preaty easy to implement it!
I will start and I will tell you when it is finished!

**Comment by JJJS777** [comment]
thank you for your reply. I have done everything so far only the receiving of packets does not work yet. do you think it could have something to do with 
```cpp
void LoraMesher::setDioActionsForScanChannel()
{
    // set the function that will be called
    // when LoRa preamble is not detected within CAD timeout period
    // or when a packet is received
    radio->setDio1Action(onReceivingTimeout);

    // set the function that will be called
    // when LoRa preamble is detected
    radio->setDio1Action(onReceive);
}

void LoraMesher::setDioActionsForReceivePacket()
{
    radio->setDio1Action(onReceive);
    radio->setDio1Action(onReceivingTimeout);
}

void LoraMesher::clearDioActions()
{
    Log.verboseln(F("Clearing callback function"));
    radio->clearDio1Action();
}
```
in [LoraMesher.cpp](https://github.com/LoRaMesher/LoRaMesher/blob/d9d923679e97ad58d79bc5f76ada02ad5e8f9ba0/src/LoraMesher.cpp#L107)? 
I changed all three ```setDio0Action```to ```setDio1Action``` because RadioLib only provides a [setDio1Action](https://jgromes.github.io/RadioLib/class_s_x126x.html#a0da667fe702d7b4aafaa4bf7e69ea40d) method for the sx1262

**Comment by Jaimi5** [comment]
Hi!

This would not work correctly. You could remove the setDioActionsForScanChannel, and remove the onReceivingTimeout.
Then in the send routine you would need to remove the startscanchannel and probably this would work.

If you require further assistance, please contact me through my email, and we will try to arrange a meeting.

**Comment by Jaimi5** [comment]
This issue has been fixed with the following pull request (#14). 
Anything else reopen the issue!

Thanks!
